src/test/java/uk/gov/pay/api/it/telephone/BuiltInValidationIT.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.pay.api.it.telephone;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.pay.api.utils.PublicAuthMockClient;
+
+import java.util.Map;
+
+public class BuiltInValidationIT extends TelephonePaymentResourceITBase {
```

Comment:
The name can be misleading. The tests are related to the usage of predefined annotations, but I wouldn't say they're built in.

Reply:
Renamed to Predefined annotations